docs/interchange.rst

```diff
@@ -55,7 +56,9 @@ sample
     Those nodes in the tree that we have obtained data from.  These are
     distinguished from other nodes by the fact that a tree sequence *must*
```

Comment:
I think that's a good definition, because we'll rely on having not-completely-specified sample nodes for recapitation. (Also, sex chromosomes, partially genotypes individuals, ...)

Huh: if you have nodes that aren't completely specified, their genotypes are specified as the ancestral state; I initially would have hoped they'd be "missing", somehow.  But gee, what else would we do here?  OK, I think this is correct, although a limitation to the "ancestral state" scheme (an alternative would be to associate a state to each root??); let's just document this behavior and move on.